server/rest/src/main/java/org/infinispan/rest/configuration/RestServerConfigurationBuilder.java

```diff
@@ -49,6 +51,11 @@ public RestServerConfigurationBuilder maxContentLength(int maxContentLength) {
       return this;
    }
 
+   public RestServerConfigurationBuilder compressionLevel(int compressLevel) {
+      this.compressionLevel = compressLevel;
```

Comment:
I think it might be a good idea to validate this input as well. Make sure it is 0-9.

Reply:
done